src/app/components/case/tasks/ChangeDueDate/ChangebleDueDate.tsx

```diff
@@ -39,8 +40,8 @@ const ChangeableDueDate: React.FC<Props> = ({ dueDate, camundaTaskId, caseId })
 
   const [, { execPost }] = useDueDate({ lazy: true })
   const onSubmit = (data: any) => {
```

Comment:
Type is `Components.Schemas.CamundaDateUpdate`

Reply:
Fixed it